protos/electricity.proto

```diff
@@ -48,6 +48,14 @@ message ClaimedEvent {
     project_origin.common.v1.Uuid allocation_id = 2;
 }
 
+message WithdrawnEvent {
```

Comment:
???? empty event?

Reply:
yes withdraw event is empty. the certificate info is in the header